package.json

```diff
@@ -47,6 +47,8 @@
     "start": "react-scripts start"
   },
   "dependencies": {
-    "popper.js": "^1.11.1"
-  }
-}
+    "popper.js": "^1.11.1",
+    "@types/react": "*"
```

Comment:
Is this should be in `devDependencies`?

Reply:
@moshfeu yep, fixed. pushed to PR.

Reply:
I hope that @tvkhoa will ever merge it 🙏